trieste/acquisition/function.py

```diff
@@ -130,6 +133,110 @@ def expected_improvement(model: ModelInterface, eta: tf.Tensor, at: QueryPoints)
     return (eta - mean) * normal.cdf(eta) + variance * normal.prob(eta)
 
 
+class MaxValueEntropySearch(SingleModelAcquisitionBuilder):
+    """
+    Builder for the max-value entropy search acquisition function (for function minimisation)
+    """
+
+    def __init__(self,search_space: SearchSpace, num_samples: int = 10, grid_size: int = 5000):
+        """
+        :param search_space: The search space over which the acquisiiton function is defined.
```

Comment:
if we use MES with the `TrustRegion` acquisition rule, the acquisition function is optimized over the trust region search space, which may be different from the global search space, so might the following be clearer? 
```suggestion
        :param search_space: The global search space over which the Bayesian optimization problem is defined.
```